db/README.md

```diff
@@ -0,0 +1,11 @@
+# Add Zip Nodes to the database
+
+You can use DB Browser for SQLite to insert zip nodes into a temp database. Let's call it zip_hazard_temp. After a successful
+bulk insert, you can extract the data into zip_hazard table which is a zip code and hazard id table with the corresponding Boolean value. 
+    
```

Comment:
I feel like we should have a package that writes the csv from pandas into SQL. I've never worked with DB Browser, but it doesn't seem like a real pipeline where I can change the csv file and run a script to update the table. Kameron and I did this in R using DBI https://db.rstudio.com/dbi/ , there should be something equivalent in Python. Let me know if you need any help.